app/utils/replace-links.js

```diff
@@ -19,6 +19,7 @@ function replaceInternalLinks(url) {
   // Map internal links to the correct route name
   return url
     .replace(/^https:\/\/emberjs.com\//, '/')
+    .replace(/^\/team$/, '/teams')
```

Comment:
A temporary solution until we update the URLs in `ember-styleguide` and make a patch release.